core/templates/services/questions-list.service.ts

```diff
@@ -57,13 +57,13 @@ export class QuestionsListService {
     this._questionSummartiesInitializedEventEmitter.emit();
   }
 
-  private _setNextQuestionsCursor(nextCursor: string): void {
-    this._nextCursorForQuestions = nextCursor;
+  private _setNextQuestionsOffset(nextOffset: number): void {
+    this._nextOffsetForQuestions = nextOffset;
   }
 
   isLastQuestionBatch(): boolean {
     return (
-      this._nextCursorForQuestions === null &&
+      this._nextOffsetForQuestions === null &&
```

Comment:
Why are we comparing it to `null`?

Reply:
Earlier the `cursor` was null when it was last page. So I updated next_offset to be null when the page was last page. So just replaced cursor with offset.

Reply:
Ah, okay, maybe add a comment about this.

Reply:
Done